shell/platform/darwin/ios/BUILD.gn

```diff
@@ -249,6 +251,30 @@ shared_library("copy_and_verify_framework_headers") {
   ]
 }
 
+test_fixtures("flutter_tests_fixtures") {
+  fixtures = []
+}
+
+ios_app("FlutterTests") {
+  testonly = true
+
+  info_plist = "FlutterTestsInfo.plist"
+
+  sources = [
+    "flutter_unittests.mm",
+  ]
+
+  libs = [
+    "Foundation.framework",
+  ]
+
+  deps = [
+    ":flutter_tests_fixtures",
+    "$flutter_root/runtime:libdart",
```

Comment:
We can stop using `flutter_root` now. If you still want to use it though, please use it consistently in rules (the next dep does not use the variable here).